bin/matrix-config.js

```diff
@@ -1,124 +1,124 @@
+
+          if (options.watch) {
+            firebase.app.onChange(target, handleResponse)
+          }
+
+        } else if (pkgs.length === 2) {
+          // get deep
+
+          var key = pkgs[1];
+          var target = pkgs[0] + '/' + key.replace(/\./g, '/');
+          debug('Firebase: '.blue, target)
+          firebase.app.get(target, handleResponse)
+
+          if (options.watch) {
+            firebase.app.onChange(target, handleResponse)
+          }
+          // matrix config base keywords=test,keyword
+        } else if (pkgs.length >= 3) {
+          var key = pkgs[1];
+          var value = pkgs[2];
+
+          //typing
+          if (pkgs.length > 4) {
+            //string
+            value = _.drop(pkgs, 2).join(' ');
+          } else {
+            // is array?
+            if (value.indexOf(',') > -1) {
+              value = value.split(',');
             }
+          }
 
-    } else if (pkgs.length === 2){
-      // get deep
+          var target = pkgs[0] + '/' + key.replace(/\./g, '/');
 
-      var key = pkgs[1];
-      var target = pkgs[0]+'/'+ key.replace(/\./g,'/');
-      debug('Firebase:'.blue, target)
-      firebase.app.get( target, handleResponse )
+          debug(target, value);
 
-      if(options.watch){
-        firebase.app.onChange( target, handleResponse )
-      }
-      // matrix config base keywords=test,keyword
-    } else if (pkgs.length >= 3){
-      var key = pkgs[1];
-      var value = pkgs[2];
-
-      //typing
-      if ( pkgs.length > 4 ){
-        //string
-        value = _.drop( pkgs, 2 ).join(' ');
-      } else {
-        // is array?
-        if ( value.indexOf(',') > -1 ){
-          value = value.split(',');
-        }
-      }
-
-      var target = pkgs[0]+'/'+ key.replace(/\./g,'/');
+          firebase.app.set(target, value);
 
-      debug(target, value);
-
-      firebase.app.set(target, value);
-
-    } else {
-      showHelp();
-    }
-});
+        } else {
+          showHelp();
+        }
+    });
 
-function handleResponse(err, app){
-  if (err) return console.error(err);
+  function handleResponse(err, app) {
+    if (err) return console.error(err);
 
-  console.log(require('util').inspect(app, {depth: 3, colors:true}));
+    console.log(require('util').inspect(app, { depth: 3, colors: true }));
 
-}
+  }
 
 
-function showHelp() {
+  function showHelp() {
 
-  console.log('\n> matrix config ¬\n');
-  console.log('\t         matrix config -', 'get device configurations'.grey)
-  console.log('\tmatrix config <appName> -', 'get application configurations'.grey)
-  console.log('\tmatrix config <appName> <key> -', 'get application configuration key'.grey)
-  console.log('\tmatrix config <appName> <key> <value> -', 'set application configuration key value'.grey)
-  console.log('\n')
-  process.exit(1);
-}
+    console.log('\n> matrix config ¬\n');
+    console.log('\t         matrix config -', i('matrix.config.help').grey)
```

Comment:
I would have preferred to have the namespace be `matrix.help.config` but that's ok for now.